docs/guide/ecosystem/documentation.md

```diff
@@ -203,25 +200,48 @@ props: {
 #### Cons
 
 - Static code samples
+- Unsupported features of VueJs
+
+#### Useful Links
+
+- [Github](https://github.com/vuese/vuese)
+- [Example of markdown-it with vuese](https://buptsteve.github.io/markdown-it-vuese/example/demo.html#props)
 
 ### Vue Design System
 
-https://github.com/viljamis/vue-design-system
+Vue design system is a combination of [vue-styleguidist](#vue-styleguidist) and [salesforce-ux/theo](https://github.com/salesforce-ux/theo). It goes by *vueds* for short. Vueds is a boilerplate for making your own design systems. It is highly opinionated. If you are starting a new project or if you don't want to think about your naming conventions simply clone the project.
 
-Vue designs system is a combination of vue-styleguidist and [salesforce-ux/theo](https://github.com/salesforce-ux/theo) that helps teams build a starter kit for their design system. It is highly opinionated. Choose it if you are starting a new project or if you don't want to think about your own naming.
+[vueds documentation](https://github.com/viljamis/vue-design-system/wiki) is a great read. Even if you are not going to use it directly, you will love to learn about design systems and how to use them.
 
-vueds documentation is a great read, even if you are not going to use it.
+#### Useful Links
 
-## Documentation Generators
+- [Example](https://vueds.com/example/)
+- [Official website](https://vueds.com/)
+- [Github](https://github.com/viljamis/vue-design-system)
 
-Automatic documentation generators scan your files and output a structured format, that is easy to either use directly or be passed to another tool.
+## Documentation Generators
 
-Such tools can scan your files for JSDoc blocks and output either a json file or full blown websites, ready to deploy.
+Automatic documentation generators scan your files and output a structured format. Some create markdown, some create html pages.
 
 ### Vuedoc
 
-https://gitlab.com/vuedoc/md
+Vuedoc is using JSDoc to build your documentation in a markdown file.
+Since it is using JSDoc it can easily be combined with styleguidist.
```

Comment:
```suggestion
Since it is using JSDoc, it can easily be combined with styleguidist.
```
How? Is this explained anywhere? If no, then we should not confuse ppl.

Reply:
Let's not, I will remove it

Reply:
Removed